.github/workflows/rack-box-continuous.yml

```diff
@@ -5,12 +5,7 @@
 
 name: RACK-in-a-Box continous build
 
-on:
-  pull_request:
-    branches: [ '*' ]
-  push:
-    branches: [ '*' ]
-    tags-ignore: [ '*' ]
+on: [push]
```

Comment:
I assume this change helps with not running multiple / redundant workflows, but how does it help?  Also, don't we still need tags-ignore?  My concern is that if we don't ignore tags when we publish a new RACK release, the new tag pushed by publishing the release will kick off all three of rack-box-continuous.yml, rack-box-release.yml, and rack-continuous.yml which still would be excessive.

Reply:
I made this change after my first push to this PR, which didn't trigger an associated CI build for some reason. Let me revert and see if that still happens...

Reply:
For context, see https://github.com/ge-high-assurance/RACK/issues/231

Reply:
Yep, looks like it's working now. Reverted.